larq/layers_base.py

```diff
@@ -14,7 +14,7 @@ def _compute_padding(stride, dilation_rate, input_size, filter_size):
     effective_filter_size = (filter_size - 1) * dilation_rate + 1
     output_size = (input_size + stride - 1) // stride
```

Comment:
This line still throws ``TypeError: unsupported operand type(s) for +: 'NoneType' and 'int'``

Reply:
OK, looks like we can also have dynamic shapes in `Layer.call`. I changed it to pass a Tensor.